Documentation/FileFormats.md

```diff
@@ -0,0 +1,58 @@
+# Serenity Formats
+
+This document summarizes all non-standard file formats, data formats, and mime-types used by Serenity.
+
+## glyph/x-fonteditor (Clipboard-only)
+
+Requires the metadata-fields, `char` (usually encoded as a string of length 1: the codepoint as a character),
+`width`, and `height` (encoded as a decimal number).
+
+The data is encoded in width times height many bytes, either 0 (clear) or 1 (set).
```

Comment:
This is a bit vague, and I'd have to look up how it works. Is it possible to give an actual example in textual representation? How are these fields stored?

Reply:
Can you specify which part is vague? Or is it the overall thing? Maybe it helps to phrase it like this?: "The glyph bitmap data is encoded …"

In general, I try to not be too specific about how it works; this document should just be an entry point to learn about which formats exist, and where to learn the details.

Reply:
> Can you specify which part is vague?

Whether the metadata is part of the payload or not. I did look at the source, and it's not - just a regular `HashMap` passed to `GUI::Clipboard::the().set_data()`. With that as context, this makes much more sense, so maybe:

> Requires the following fields to be present in the clipboard metadata `HashMap`: ...

?

Reply:
Right, in this document I should not assume that the reader already knows how the clipboard works.